lib/builder.dart

```diff
@@ -22,5 +24,16 @@ class NuvigatorGenerator extends GeneratorForAnnotation<NuRouterAnnotation> {
   }
 }
 
+class NextGenerator extends GeneratorForAnnotation<ModuleArguments> {
```

Comment:
We created a new generator for the next/api. 

Should we continue to use the same generator or creating a new one makes more sense?

Reply:
Good point, if we can use both simultaneous, I think it's better to keep separate to avoid loading the other generator if we choose not to use it